source/overview/corba.rst

```diff
@@ -0,0 +1,32 @@
+10 things you should know about CORBA
+=====================================
+
+1.  ***You don’t need to know CORBA to work with TANGO***
+
+2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest
+        **B**\ roker **A**\ rchitecture and it is a standard defined by
+        the\ ` <https://en.wikipedia.org/wiki/Object_Management_Group>`__\ Object
+        Management Group (OMG)
```

Comment:
Hi @Matteo04052017, please make points indentation correct, like below:
```
2.  CORBA is the acronym for **C**\ ommon **O**\ bject **R**\ equest
    **B**\ roker **A**\ rchitecture and it is a standard defined by
    the\ ` <https://en.wikipedia.org/wiki/Object_Management_Group>`__\ Object
    Management Group (OMG)
``` 
(Subsequent lines of the same point should be spaced with such number of space so it starts below latter of the first line)